src/main/java/de/uniba/dsg/serverless/model/PerformanceData.java

```diff
@@ -43,20 +50,22 @@ public PerformanceData(String functionName, String logStream, String requestId,
 	@Override
 	public String toString() {
 		return "PerformanceData [functionName=" + functionName + ", logStream=" + logStream + ", requestId=" + requestId
-				+ ", startTime=" + startTime + ", preciseDuration=" + preciseDuration + ", billedDuration="
-				+ billedDuration + ", memorySize=" + memorySize + ", memoryUsed=" + memoryUsed + "]";
+				+ ", startTime=" + startTime + ", endTime=" + endTime + ", startupDuration=" + startupDuration
+				+ ", preciseDuration=" + preciseDuration + ", billedDuration=" + billedDuration + ", memorySize="
+				+ memorySize + ", memoryUsed=" + memoryUsed + "]";
 	}
 
 	public static String getCSVMetadata() {
 		return "FunctionName" + CSV_SEPARATOR + "LogStream" + CSV_SEPARATOR + "RequestID" + CSV_SEPARATOR + "StartTime"
```

Comment:
A possiblity to the present start-up handling is to use the FunctionName and insert as a functionName "StartUp" or a similar text, because the connection between the first execution and the start-up time is somehow wrong in azure...

Opinions? This approach resolves the todo in AzureLogHandler.